### PR TITLE
#155124480 Heroku deployment 

### PR DESCRIPTION
**Pull request by shakirandagire**
#### What does this PR do?
Heroku deployment

#### Description of Task to be completed?
Ensure the project is deployed on heroku for staging.

#### How should this be manually tested?
https://wg-logros.herokuapp.com

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[155124480](https://www.pivotaltracker.com/story/show/155124480)

### CODE REVIEW

**Review by muchai-mercy — CHANGES_REQUESTED** [review]
@shakirandagire Did you make 1068 files changes. Not sure where to start reviewing

**Comment by shakirandagire** [comment]
Let me work on the gitignore and push to reduce the number of files changed

**Comment by shakirandagire** [comment]
I have worked on it and now the files have reduced to 9 changed files

File: wger/settings_global.py [hunk 1]
```diff
@@ -317,13 +319,14 @@
...
 #
-STATIC_ROOT = ''
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATIC_URL = '/static/'
 
 SOCIAL_AUTH_FACEBOOK_KEY = '197453420843149'
 SOCIAL_AUTH_FACEBOOK_SECRET = 'a4fcf058e205ee880313053339ce3924'
 
 
+# STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```
Comment (muchai-mercy) [inline wger/settings_global.py]: Should this be commented out? If not in use, we can clear it out.

File: .gitignore [hunk 2]
```diff
@@ -65,3 +64,5 @@ node_modules
 .DS_Store
 venv-django
 .vscode/
+# .env
```
Comment (muchai-mercy) [inline .gitignore]: Do we need this? @shakirandagire given you already added `.env`?

Comment (shakirandagire) [reply .gitignore]: @Mercy-Muchai I had put it as a title. But let me remove it since its not necessary

Comment (shakirandagire) [reply .gitignore]: @Mercy-Muchai I removed the # .env from gitignore file

**Review by kevinsamoei — APPROVED** [review]
LGTM

**Review by JoyyToo — APPROVED** [review]
LGTM

**Comment by kevinsamoei** [comment]
@Mercy-Muchai The settings file has been updated. The unused code has been removed. Review the changes.

**Review by muchai-mercy — APPROVED** [review]
LGTM

**Pull request merged**